flink-runtime-web/web-dashboard/src/app/interfaces/task-manager.ts

```diff
@@ -36,6 +36,7 @@ export interface TaskmanagersItemInterface {
   path: string;
   dataPort: number;
   timeSinceLastHeartbeat: number;
+  milliSecondsSinceLastHeartBeat: number;
```

Comment:
you can create a new interface, put and use the interface in the share component mentioned below
 
```typescript
TaskmanagersItemCalInterface extends TaskmanagersItemInterface {
  milliSecondsSinceLastHeartBeat: number;
}
```